dev/com.ibm.ws.microprofile.faulttolerance.2.1_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -16,6 +16,7 @@
     <name>MicroProfile Fault Tolerance TCK Runner TCK Module</name>
 
     <properties>
+    	<microprofile.faulttolerance.version>2.1</microprofile.faulttolerance.version>
```

Comment:
Not sure why you've added this. It looks like you're creating a property, but you haven't referenced it anywhere. I think I'd just take it out.

Reply:
This was in an attempt to fix the build failures. Upon further investigation, I have removed this line.